cristin-project/src/main/java/no/unit/nva/cristin/projects/query/QueryCristinProjectHandler.java

```diff
@@ -24,9 +20,8 @@
  */
 public class QueryCristinProjectHandler extends CristinQueryHandler<Void, SearchResponse<NvaProject>> {
 
-    public static final Set<String> VALID_QUERY_PARAMETERS = QueryParameterKey.VALID_QUERY_PARAMETER_NVA_KEYS;
     public static final QueryParameterKey[] REQUIRED_QUERY_PARAMETER =
-        {PATH_PROJECT, PAGE_CURRENT, PAGE_ITEMS_PER_PAGE, LANGUAGE};
+        {PATH_PROJECT, PAGE_CURRENT, PAGE_ITEMS_PER_PAGE};
```

Comment:
PAGE_CURRENT, PAGE_ITEMS_PER_PAGE, LANGUAGE, all have default values, that will be populated, if they haven't any values set. But if they are not set to required, cristinQuery will not add them,